server/export.go

```diff
@@ -13,9 +13,13 @@ import (
 	"path"
 )
 
+const (
+	flagHeight = "height"
```

Comment:
This flag also lives in the client package: https://github.com/cosmos/cosmos-sdk/blob/develop/client/flags.go#L23

Reply:
Indeed - I don't know if we want to import the client package from the server package though.

Maybe we should have a common "flags" package.